dev/abft.md

```diff
@@ -44,17 +44,26 @@ All integers described in this document are unsigned.
 Parameters
 ==========
 
+\newcommand \FilterTimeout {\mathrm{FilterTimeout}}
+
 The protocol is parameterized by the following constants:
 
- - $\lambda, \lambda_f, \Lambda$ are values representing durations of time.
+ - $\lambda, \lambda_0, \lambda_f, \Lambda$ are values representing durations of time.
  - $\delta_s, \delta_r$ are positive integers (the "seed lookback" and
    "seed refresh interval").
 
 For convenience, we define $\delta_b$ (the "balance lookback") to be
 $2\delta_s\delta_r$.
 
+We define $\FilterTimeout(p)$ on period $p$ as follows:
+
+  - If $p = 0$:
+    - $\FilterTimeout(p) = 2\lambda_0$
+  - If $p \ne 0$:
+    - $\FilterTimeout(p) = 2\lambda$
+
 Algorand v1 sets $\delta_s = 2$, $\delta_r = 80$, $\lambda = 2$ seconds,
-$\lambda_f = 5$ minutes, and $\Lambda = 17$ seconds.
+$\lambda_0 = 1.7$ seconds, $\lambda_f = 5$ minutes, and $\Lambda = 17$ seconds.
```

Comment:
What does `v1` means in `Algorand v1`?

Reply:
Good question. Maybe we should remove v1?